lib/profilefunctions.js

```diff
@@ -216,18 +229,27 @@ function init(profileData) {
 
     var basal = profile.getBasal(time, spec_profile);
     var tempbasal = basal;
+    var combobolusbasal = 0;
+    var totalbasal = basal;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['totalbasal' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1498320447/issues/source?bid=2452379&fileBranchId=2998783#l233)